numpy/linalg/umath_linalg.c.src

```diff
@@ -651,39 +651,39 @@ CDOUBLE_mulc(DOUBLECOMPLEX_t op1, DOUBLECOMPLEX_t op2)
     return result;
 }
 
-static inline void
+static NPY_INLINE void
 print_FLOAT(npy_float s)
 {
     TRACE_TXT(" %8.4f", s);
 }
-static inline void
+static NPY_INLINE void
 print_DOUBLE(npy_double d)
 {
     TRACE_TXT(" %10.6f", d);
 }
-static inline void
+static NPY_INLINE void
 print_CFLOAT(npy_cfloat c)
 {
     float* c_parts = (float*)&c;
```

Comment:
Either way, I think I plan to leave this PR at this point (modulo a squash, perhaps). If someone else wants to push some extra cleanup to this branch, they're welcome to. Or maybe it's worth merging as is.